### DISCUSSION TITLE
Accessing Snowflake via SSO

### DISCUSSION THREAD

**Discussion by TNakagawa3q** [dbeaver/dbeaver#13851] [Q&A] [answered]
#### System information: 
- DBeaver version 21.1.5

#### Connection specification:
- Snowflake
- JDBC

#### Describe the problem you're observing:
We have Snowflake SSO configured to use Google as our IDprovider. Logging into Snowflake still seems to require UN/PW. Is there a way around this so that our users don't need to keep UN/PW and just use SSO?

**Comment by uslss** [comment, marked as answer]
probably you should try to change authenticator to externalbrowser (#3902)
https://docs.snowflake.com/en/user-guide/jdbc-configure.html
